Steps4Impact.xcodeproj/project.pbxproj

```diff
@@ -794,7 +794,7 @@
 					};
 					269E34C51E765828000726C2 = {
 						CreatedOnToolsVersion = 8.2.1;
-						DevelopmentTeam = 6SQ8F32998;
+						DevelopmentTeam = 479QYREFPY;
```

Comment:
please don't commit changes to project.pbxproj that effect build settings like dev team or bundle ids